errors/parameter_errors_test.go

```diff
@@ -5,23 +5,25 @@ package errors
 
 import (
 	"context"
+	"testing"
+
 	"github.com/pb33f/libopenapi/datamodel/high/base"
+	"github.com/pb33f/libopenapi/datamodel/low"
 	"github.com/pb33f/libopenapi/orderedmap"
```

Comment:
I would expect this package and everything under pb33f to be regrouped.

I'm raising it again as you resolved.
But I'm finr if you disagree with me 😁

Localmodule might have not been enough.

Reply:
Wdyt about distinct blocks for localmodule and other pb33f modules?

Reply:
That's the last solution I suggested here

https://github.com/pb33f/libopenapi-validator/pull/101#discussion_r1832532789

So of course, I'm OK with it.

But please note, I'm a random Gopher. I'm not a maintainer of this repository.

Maybe @daveshanley will have another point of view, but whatever the gci config. I'm looking for enforcing the imports, as it's currently not consistent

Reply:
I have no preference at all.